internal/statestore/redis.go

```diff
@@ -529,13 +534,36 @@ func (rb *redisBackend) UpdateAssignments(ctx context.Context, req *pb.AssignTic
 			return nil, status.Errorf(codes.Internal, "failed to marshal ticket %s", ticket.GetId())
 		}
 
-		cmds = append(cmds, ticket.GetId(), ticketByte)
+		err = redisConn.Send("SET", ticket.Id, ticketByte, "PX", int64(assignmentTimeout), "XX")
+		if err != nil {
+			return nil, errors.Wrap(err, "error sending ticket assignment set")
+		}
 	}
 
-	_, err = redisConn.Do("MSET", cmds...)
```

Comment:
Wasnt moving to MSET an optimization? Why move back to EXEC? and if there are reasons, then may be do that consistently for other patterns in the storage layer?

https://github.com/googleforgames/open-match/commit/3de052279b5a725c2980851a29056b8a45ead401

Reply:
Redis does not provide a function which both sets multiple values, and also allows setting the expiration at the same time.  As far as I can see there's only setting expiration one ticket at a time.  So it's fewer commands to set them with their expiration.  As a bonus, it's possible to say only set they key if it exists, which prevents a race condition if the ticket is deleted between reading and writing.